test/web/10-sessionService.js

```diff
@@ -0,0 +1,138 @@
+      ({account, totp, email, password} = await createAccount(
+        mockData.accounts.sessionService));
+    });
+    beforeEach(async function() {
+      session = null;
+      await login({email, password, totp});
+    });
+    afterEach(async function() {
+      await sessionService.logout();
+    });
+    it('should get a session with an account', async () => {
+      let err;
+      try {
+        session = await sessionService.get();
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      should.exist(session);
+      session.should.be.an('object');
+      const keys = Object.keys(session);
+      // an authenticated session has data
+      keys.should.deep.equal(['account']);
+      session.account.should.be.an('object');
+      session.account.should.have.property('id');
+      session.account.id.should.equal(account.id);
+    });
+    it('should logout a session', async () => {
+      let err;
+      try {
+        session = await sessionService.get();
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      should.exist(session);
+      session.should.be.an('object');
+      let keys = Object.keys(session);
+      // an authenticated session has data
+      keys.should.deep.equal(['account']);
+      session.account.should.be.an('object');
+      session.account.should.have.property('id');
+      session.account.id.should.equal(account.id);
+      await sessionService.logout();
+      session = await sessionService.get();
+      keys = Object.keys(session);
+      // an unauthenticated session has no data
+      keys.should.deep.equal([]);
+    });
+    it('should expire after 1 second', async function() {
+      let err;
+      try {
+        session = await sessionService.get();
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      should.exist(session);
+      session.should.be.an('object');
+      let keys = Object.keys(session);
+      // an authenticated session has data
+      keys.should.deep.equal(['account']);
+      session.account.should.be.an('object');
+      session.account.should.have.property('id');
+      session.account.id.should.equal(account.id);
+      await delay(2000);
+      session = await sessionService.get();
+      keys = Object.keys(session);
+      // an unauthenticated session has no data
+      keys.should.deep.equal([]);
+    });
+    it('should refresh on get', async function() {
+      let err;
+      try {
+        session = await sessionService.get();
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      should.exist(session);
+      session.should.be.an('object');
+      let keys = Object.keys(session);
+      // an authenticated session has data
+      keys.should.deep.equal(['account']);
+      session.account.should.be.an('object');
+      session.account.should.have.property('id');
+      session.account.id.should.equal(account.id);
+      for(let i = 0; i < 4; i++) {
```

Comment:
I think 5 iterations here is more to the point.  Also, please add a comment that you are demonstrating that the session would have expired at 1000ms if the refresh were not working properly.  Just provide the reader more information about the aims of the test and how your are accomplishing them.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-web-session/pull/9/commits/57c346b2c53b1f3110f15b5356fc0f937a0cbaec